noir_stdlib/src/array/quicksort.nr

```diff
@@ -1,8 +1,8 @@
-unconstrained fn partition<T, Env, let N: u32>(
+unconstrained fn partition<T, let N: u32>(
     arr: &mut [T; N],
     low: u32,
     high: u32,
-    sortfn: fn[Env](T, T) -> bool,
```

Comment:
Same here

Reply:
Leftover from debugging blocking issue: I've restore it